website/templates/project/addon/widget.mako

```diff
@@ -1,5 +1,5 @@
-<div class="panel panel-default" name="${short_name}">
-
+% if complete or 'write' in user['permissions']:
```

Comment:
Contributors potentially have `read` permissions. But because your `if` is an `or`, it's showing if it's complete or if it's not complete but the user has write permissions. Proooobably this should just be `if complete`. However… (see comment on the code farther down that I'm about to put in)